Makefile

```diff
@@ -147,10 +147,10 @@ client-unit-test: client-setup-env ## Run client unit tests
 client-integration-test: client-setup-env ## Run client integration tests
 	@echo "--- Starting client integration tests ---"
 	@echo "Ensuring Docker Compose services are stopped and removed..."
-	@$(CONTAINER_TOOL) compose -f $(PYTHON_CLIENT_DIR)/docker-compose.yml kill || true # `|| true` prevents make from failing if containers don't exist
```

Comment:
CI on main is currently failing:
```
--- Starting client integration tests ---
Ensuring Docker Compose services are stopped and removed...
bash: line 1: compose: command not found
bash: line 1: compose: command not found
Bringing up Docker Compose services in detached mode...
bash: line 1: compose: command not found
make: *** [Makefile:153: client-integration-test] Error 127
```
https://github.com/apache/polaris/actions/runs/16822814767/job/47652969513